frontend/src/model/beta/overview/TemplatePage.tsx

```diff
@@ -68,9 +72,9 @@ export default function TemplatePage({ model }: TemplatePageProps) {
                 Create from scratch
               </Typography>
               <Typography variant='body1'>Create a model from scratch using a predefined schema.</Typography>
-              <Button variant='contained' onClick={handleCreateFromScratchClick}>
+              <LoadingButton variant='contained' onClick={handleCreateFromScratchClick} loading={loading}>
```

Comment:
This button also isn't needed, there's no API interaction here so we're not waiting for anything. All we're doing is changing the route (which just changes which page the user is on).